stanford/stage-ssh-config

```diff
@@ -1,20 +1,8 @@
-#### edx-stage VPC 
-
-Host 54.241.183.3
-#Host vpc-jumpbox
-    HostName 54.241.183.3
-    User ubuntu
-    ForwardAgent yes
-
-Host *.us-west-1.compute.internal
-    User ubuntu
-    ForwardAgent yes
-    ProxyCommand ssh -W %h:%p ubuntu@54.241.183.3
-
 Host *
     ForwardAgent yes
     SendEnv LANG LC_*
     HashKnownHosts yes
     GSSAPIAuthentication yes
     GSSAPIDelegateCredentials no
-
+    User ubuntu
+    ProxyCommand ssh -W %h:%p ubuntu@54.241.183.3
```

Comment:
We're pulling these two values in from the `*.internal` stanza above; they can appear globally in the stage ssh context.